docs/changes/cosmology/12279.feature.rst

```diff
@@ -0,0 +1,2 @@
+Register yaml into Cosmology's ``to/from_format`` I/O, allowing
+a Cosmology instance to be parsed from or converted to a yaml string.
```

Comment:
YAML (the format) is always all caps. So maybe `Register "yaml" into Cosmology's ... to a YAML string.`

Reply:
There should be a similar changelog entry for the `io.misc` sub-package indicating that built-in Cosmology subclasses can now be converted to/from YAML with dump and load from astropy.io.misc.yaml.

Reply:
👍

Reply:
Done.